src/adapt_tests.cpp

```diff
@@ -79,9 +79,9 @@ void test_adapt(parser const &problem, std::filesystem::path gold_base)
   };
 
   auto const gold_coarse_table =
-      fk::matrix<int>(read_matrix_from_txt_file(table_coarse_path));
+      fk::matrix<int>(read_matrix_from_txt_file<int>(table_coarse_path));
```

Comment:
```suggestion
      read_matrix_from_txt_file<int>(table_coarse_path);
```
We should skip the additional move constructor.

Reply:
I missed a spot?

Reply:
We don't need to call the extra move constructor
https://github.com/project-asgard/asgard/blob/2d8a205ce2a63b145bba81d31f934373471ebed7/src/tensors.hpp#L2066